.go-version

```diff
@@ -0,0 +1 @@
+golang 1.17.6
```

Comment:
Why do we need `golang` prefix in `.go-version`? I think just putting a version should work.